libs/blocks/table/table.js

```diff
@@ -373,11 +372,11 @@ function applyStylesBasedOnScreenSize(table, originTable) {
       if (filters[0] > filters[1]) {
         if (isMerch) {
           rows.forEach((row) => {
-            row.querySelector('.col:not(.section-row-title)').style.order = 1;
+            row.querySelector('.col:not(.section-row-title)').classList.add('force-last');
```

Comment:
I think the `{` can be removed for both `forEach` lines

Reply:
Removed them